### ISSUE TITLE
Carbon component ComposedModal fails a11y testing

### ISSUE THREAD

**Issue by siddarthpatel** [carbon-design-system/carbon#3103]
## Detailed description

> What version of the Carbon Design System are you using?
  Using `carbon-components-react` version `6.85.0` and `carbon-components` version `9.92.0`

> What did you expect to happen?
The tests to pass without any accessibility issues

> What happened instead?

Accessibility tests for carbon component `ComposedModal` fails. Since the `aria-label` for the div `bx--modal-container` is not set.  Now that `bx--modal` is a part of `ComposedModal`. I can't manually set the aria-label for it. Passing an aria-label to `ComposedModal` does not help either. Is there some other prop or something I can use for this? I don't see anything in the documentation

<img width="814" alt="Screen Shot 2019-06-18 at 10 41 58 AM" src="https://user-images.githubusercontent.com/17713495/59709685-587a9180-91d5-11e9-9096-146cf88b27fa.png">

**Comment by shixiedesign** [comment]
FYI @dakahn

**Comment by asudoh** [comment]
The element doesn't seems to have `role="region"` in our latest code. So wanted to see what version of our library is in use. I thought we have an issue template to describe the version in use?

**Comment by dakahn** [comment]
Yes, @siddarthpatel please go back and complete the issue template. Thank you 🙏

**Comment by siddarthpatel** [comment]
@asudoh The `carbon-components-react` version is `6.85.0`. Does this mean it is fixed in later versions? If so what version? thank you

**Comment by asudoh** [comment]
Thanks. `6.85.0` doesn't seem to have `role="region"`, either: https://github.com/carbon-design-system/carbon-components-react/blob/v6.85.0/src/components/ComposedModal/ComposedModal.js#L246

That said, you want to create a reduced case that shows the problem?
